overcast/django/apps/buildsvc/pkgbuild/__init__.py

```diff
@@ -13,12 +14,14 @@
 from ..models import BuildRecord
 
 class PackageBuilder(object):
-    def __init__(self, basedir, package_source, build_record):
+    def __init__(self, basedir, package_source, build_record,
+                 docker_build=True):
```

Comment:
I don't think this should be conditional.

What's the use case for non-dockerized builds? I think it will bitrot almost immediately. Everything is still moving quite quickly (and once we start having real users, it'll move even faster), and a build method that noone is going to use will just slow down our progress.